### PR TITLE
Breakfast menu

### PR DESCRIPTION
**Pull request by KOngTNP**
add Breakfast menu and prices list.

### CODE REVIEW

**Comment by Chayapol-c** [comment]
Don't forget to add your name to README.md

File: README.md [hunk 1]
```diff
@@ -1,18 +1,10 @@
...
+[breakfast menu](Menu.md)
 
 ## About Us
 
-**TODO**: Each person adds himself to this list; **not** one person adds everyone
-
 | Name      | Role      | Github   |
 |:----------|-----------|----------|
-| Your name | Your menu | @your_github_id |
+|Tanapol Buangam|Breakfast menu| KOngTNP |
```
Comment (KOngTNP) [inline README.md:9]: add name, role and GitHub id

File: README.md [hunk 2]
```diff
@@ -1,18 +1,10 @@
 # KU Cafe
 
-**TODO**: add hyperlinks and format these one per line. Remove this TODO.
-
-Breakfast Menu
-Lunch Menu
-Dinner Menu
-Night Food Menu
-Beverages
+[breakfast menu](Menu.md)
```
Comment (KOngTNP) [inline README.md:3]: add hyperlinks

**Comment by KOngTNP** [comment]
> Don't forget to add your name to README.md

Done

**Comment by theethaj** [comment]
Please add your link to your Github profile.

**Comment by KOngTNP** [comment]
> Please add your link to your Github profile.

Done

File: Menu.md [hunk 3]
```diff
@@ -4,14 +4,12 @@
...
 |:---------------------------------------|------:|
-| nothing yet                            |  0.0  |
-
-## Lunch Menu
-
-Coming soon.
-
-etc.
-
----
```
Comment (kidstylex) [inline Menu.md:15]: THB

Comment (KOngTNP) [reply Menu.md:15]: done

**Comment by theethaj** [comment]
Please change link name from "breakfast menu" to "Breakfast Menu"

**Comment by KOngTNP** [comment]
> Please change link name from "breakfast menu" to "Breakfast Menu"

done

**Comment by theethaj** [comment]
It's all okay.

**Pull request merged**